src/lib/Functions/handlePaste.ts

```diff
@@ -31,7 +43,7 @@ export function handlePaste(event: ClipboardEvent, state: State): State {
             tableRows[ri].children[ci].getAttribute("data-reactgrid");
           const data = rawData && JSON.parse(rawData);
           const text = tableRows[ri].children[ci].innerHTML;
-          row.push(data ? data : { type: "text", text, value: parseFloat(text) });
+          row.push(data ? data : { type: "text", text, value: parseFloatLocale(text) });
```

Comment:
I also updated the function name to "localeParseFloat" which I think is more appropriate.
```suggestion
          row.push(data ? data : { type: "text", text, value: localeParseFloat(text) });
```